src/utils/ar_data_price_regression_estimator.ts

```diff
@@ -119,7 +117,14 @@ export class ARDataPriceRegressionEstimator extends AbstractARDataPriceAndCapaci
 		}
 
 		// Return 0 if winston price given does not cover the base winston price for a transaction
-		return Math.max(0, (winston - this.predictor.baseWinstonPrice()) / this.predictor.marginalWinstonPrice());
+		// TODO: Is number sufficient here vs. BigNumber?
```

Comment:
Do you mean for the returned ByteCount?
`MAX_SAFE_INTEGER` bytes to GB = `9007199.254740990698` GB

Reply:
Well the regression package uses number rather than BigNumber, so any input data you give it, or outputs it could predict, would have to be in the number range.